q2_alignment/_filter.py

```diff
@@ -79,4 +79,19 @@ def mask(alignment: skbio.TabularMSA, max_gap_frequency: float=0.05,
                                                    min_conservation)
     combined_mask = gap_mask & conservation_mask
     # apply the mask and return the resulting alignment
-    return _apply_mask(alignment, combined_mask)
+    result = _apply_mask(alignment, combined_mask)
+
+    if result.shape[1] == 0:
```

Comment:
Minor and not necessary for you to address if you don't want to: you can use `result.shape.position == 0` for more self-documenting code 🙆